src/actions/web3.ts

```diff
@@ -24,5 +26,9 @@ export function getWeb3Provider(): JsonRpcProvider {
 
 
 export async function getWeb3Signer(): Promise<Web3Provider> {
-    return new providers.Web3Provider(await web3Modal.connect())
+    const provider = new providers.Web3Provider(await web3Modal.connect())
+    if(provider.network.chainId !== 69) {
+      process.exit()
```

Comment:
```suggestion
      throw new Error("Invalid network")
```
We are in browser so we don't wanna exit :)

Reply:
{\\__/}
( • . •)
/ > `<3`

@mpetrunic had any chance to test it?